### DISCUSSION TITLE
it is very sad that theme ui have no support to use local or custom fonts.

### DISCUSSION THREAD

**Discussion by MohammadSahabuddin** [system-ui/theme-ui#1967] [General] [answered]
I have try to add custom font to my next js project but I am disappointed by seeing that there have no way to add custom font in theme-ui. I have read the documentation but I didn't find any solution.

**Comment by fcisio** [comment, marked as answer]
Hi @MohammadSahabuddin 
Font loading is not something that is usually handled by libraries like this one. How you load a font is gonna be specific to the type of font (hosted, local) and your environment (Next, Nuxt, CRA, Gatsby, ...).

This is because a lot goes into loading, preloading, caching, preventing FOUT...
This thread might be useful to you [Preload custom font with Next.js](https://stackoverflow.com/questions/63023946/preload-custom-font-with-next-js) 

What you are looking for is probably something like this [Web Font Loader](https://github.com/typekit/webfontloader)

> Once you have set up your font specific to your tools, you can simply use the font-family name in the theme.
